### DISCUSSION TITLE
There seems to be a bug with death music. (osu!lazer)

### DISCUSSION THREAD

**Discussion by PostreslesCODE** [ppy/osu#22351] [Q&A] [answered]
So when you die you get the death music right? (I think that depends on the skin)  But right now in the new version it just keeps on playing when in the menu or even on a different map, just keeps playing for like a min and its really annoying.

https://user-images.githubusercontent.com/78700829/213918053-f8df2a6b-7f38-4f70-b6ed-48a7b525d127.mp4

**Comment by mk56-spn** [comment, marked as answer]
This is known and is being addressed.

**Reply by bdach** [reply]
by https://github.com/ppy/osu/pull/22339, specifically
